MapboxGLAndroidSDKTestApp/src/androidTest/java/com/mapbox/mapboxsdk/location/LocationComponentTest.kt

```diff
@@ -1530,6 +1531,122 @@ class LocationComponentTest : EspressoTest() {
     executeComponentTest(componentAction)
   }
 
+  @Test
```

Comment:
missing a cancelation test

Reply:
Ah those cancelation tests were already ignored because they are flaky. Ignore this comment as well in this case :upside_down_face: